concent_api/verifier/tasks.py

```diff
@@ -40,6 +41,12 @@
 logger = logging.getLogger(__name__)
 
 
+def check_if_sizes_in_pixels_are_equal(image1: ndarray, image2: ndarray) -> bool:
+    shape1 = image1.shape
+    shape2 = image2.shape
+    return True if shape1[0] == shape2[0] and shape1[1] == shape2[1] else False
```

Comment:
this could be simplified to: `return image1.shape == image2.shape`

Reply:
shape is a tuple with image:
- `width`, `height` and `channels` and has 3 variables if image is in color 
or 
- `width`, `height`and has 2 variables if image is grayscale.

Now i only compare shapes. If color will be as well (i think it should be) it will be replaced with your suggestion.

Reply:
if color is not to be taken into account, it could be still simplified:
`return shape1[0] == shape2[0] and shape1[1] == shape2[1]`

Reply:
According to @cameel 's comment, color will be check as well. Changed with @dybi suggestion